framework/db/sqlite/Schema.php

```
@@ -123,15 +189,19 @@ protected function loadTablePrimaryKey($tableName)
-        $foreignKeys = $this->db->createCommand('PRAGMA FOREIGN_KEY_LIST (' . $this->quoteValue($tableName) . ')')->queryAll();
+        $resolvedName = $this->resolveTableName($tableName);
+        $foreignKeys = $this->db->createCommand(
+            $this->pragma('FOREIGN_KEY_LIST', $resolvedName->name, $resolvedName->schemaName)
+        )->queryAll();
         $foreignKeys = $this->normalizePdoRowKeyCase($foreignKeys, true);
         $foreignKeys = ArrayHelper::index($foreignKeys, null, 'table');
         ArrayHelper::multisort($foreignKeys, 'seq', SORT_ASC, SORT_NUMERIC);
         $result = [];
         foreach ($foreignKeys as $table => $foreignKey) {
             $result[] = new ForeignKeyConstraint([
                 'columnNames' => ArrayHelper::getColumn($foreignKey, 'from'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\ArrayHelper' in method 'loadTableForeignKeys'. (undefined)

(StaticAccess)